ocs_ci/ocs/md_blow.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import threading
+from time import sleep
+
+from ocs_ci.ocs.resources.pod import get_noobaa_core_pod, get_noobaa_db_pod
+from ocs_ci.ocs.resources.mcg import MCG
+
+logger = logging.getLogger(__name__)
+
+
+class MdBlow(object):
+    """
+    Upload objects directly into DB using MD Blow script present in core pod
+    """
+
+    def __init__(self):
+        """
+        Assign default values to its parameter and patch required lines in core pod
+        """
+        self.db_percentage = None
+        self.obj_count = 1000
+        self.concurrency = 50
+        self.chunks = 1
+        self.chunk_size = 200
+        mcg_obj = MCG()
+        self.creds = mcg_obj.get_noobaa_admin_credentials_from_secret()
+        self.email = self.creds["email"]
+        self.password = self.creds["password"]
+        self.noobaa_core_pod = get_noobaa_core_pod()
+        self.noobaa_db_pod = get_noobaa_db_pod()
+
+    def patch_lines_in_core_pod(self):
+        """
+        patch required lines in script to avoid failures
+        seed below line in js file at line no 32
+        await require('../server/system_services/master_key_manager.js').get_instance().load_root_keys_from_mount();
+        """
```

Comment:
why can't we have the updated script made available somewhere and access it instead of modifying the script everytime?